core/bug_api.php

```diff
@@ -795,11 +795,16 @@ function update( $p_update_extended = false, $p_bypass_mail = false ) {
  * @return array returns an array representing the bug row if bug exists
  * @access public
  */
-function bug_cache_database_result( array $p_bug_database_result, array $p_stats = null ) {
+function bug_cache_database_result( array $p_bug_database_result, $p_stats = null ) {
```

Comment:
You should also update the phpDoc block to reflect the change in function signature.